webapp/src/test/testBlockFactory.ts

```diff
@@ -112,14 +112,14 @@ class TestBlockFactory {
         return view
     }
 
-    static createCard(board?: Board): Card {
+    static createCard(board?: Board, limited = false): Card {
         const card = createCard()
         card.parentId = board ? board.id : 'parent'
         card.rootId = board ? board.rootId : 'root'
         card.title = 'title'
         card.fields.icon = 'i'
         card.fields.properties.property1 = 'value1'
-
+        card.limited = limited
```

Comment:
Same question as my other comments, can it just be passed through?

Reply:
Done